plugins/fileservices/fileservices.cpp

```diff
@@ -523,31 +523,63 @@ FILESERVICES_API void FILESERVICES_CALL fsSetReadOnly(ICodeContext *ctx, const c
     throw error.getClear();
 }
 
-
-FILESERVICES_API void FILESERVICES_CALL fsRenameLogicalFile(ICodeContext *ctx, const char *oldname, const char *newname)
+FILESERVICES_API void FILESERVICES_CALL implementRenameLogicalFile(ICodeContext *ctx, const char *oldname, const char *newname, const bool overwrite)
 {
     StringBuffer lfn, nlfn;
```

Comment:
pedantic/not new: but would be slightly clearer if lfn was olfn, or these were oldLfn/newLfn or similar.

Reply:
Changed